### DISCUSSION TITLE
Is it possible to use individual @unocss/* packages without the default unocss package?

### DISCUSSION THREAD

**Discussion by bstro** [unocss/unocss#2904] [Q&A] [answered]
I am building a simple Preact app that renders out to a statically generated site. I am trying to keep my `node_modules` directory as absolutely minimal as possible, and so I am evaluating packages partially on how many dependencies they add to my `node_modules` folder. I'm using PostCSS to do some processing on CSS in my pipeline, and my app also uses Tailwind classes. Since Tailwind installs a massive amount of additional libraries, I'd really like to find something smaller.

That said, I'm wondering about how Unocss is put together. More specifically, I'd love to know if there's a coherent way to just use `@unocss/core` along with `@unocss/preset-wind`, or if those two are actually dependant on the main `unocss` package. The latter installs a fair amount of dev dependencies, and the former very few, so it would potentially be an attractive option for me, especially if it were possible to do so easily with PostCSS.

Thanks!

**Comment by zyyv** [comment, marked as answer]
Check https://github.com/unocss/unocss/blob/main/packages/unocss/src/index.ts
The `unocss` main package is equivalent to a whole forwarding, it is not required. You can install dependencies on demand to use.
